src/host/getset.cpp

```diff
@@ -835,18 +839,20 @@ void ApiRoutines::GetLargestConsoleWindowSizeImpl(const SCREEN_INFORMATION& cont
         //      this scenario is highly unlikely, and we can reasonably do this
         //      on their behalf.
         // see MSFT:19853701
+
+        // use context (not buffer) here because alternate buffer may not be aware of VT mode
         if (context.InVTMode())

```

Comment:
```suggestion
        if (buffer.InVTMode())
```

Reply:
Okay the spacing looks weird in the GH preview, just change this locally to be safe